### PR TITLE
add BulkGraviton cards for HGCal TDR

### PR DESCRIPTION
**Pull request by jalimena**
Needed for the jet substructure study within the FNP group of the UPSG, for the HGCal TDR

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/pre2017/14TeV/BulkGraviton_VBF_WW_inclu/BulkGraviton_VBF_WW_inclu_narrow_M2000/BulkGraviton_VBF_WW_inclu_narrow_M2000_proc_card.dat [hunk 1]
```diff
@@ -0,0 +1,13 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set gauge unitary
+set complex_mass_scheme False
+import model RS_bulk_ktilda
+define p = u c d s u~ c~ d~ s~
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/pre2017/14TeV/BulkGraviton_VBF_WW_inclu/BulkGraviton_VBF_WW_inclu_narrow_M2000/BulkGraviton_VBF_WW_inclu_narrow_M2000_proc_card.dat]: remove proton and jet definition as is picked up automatically based on the import model.
by the way, is this really a 4F without gluon?!?

File: bin/MadGraph5_aMCatNLO/cards/production/pre2017/14TeV/BulkGraviton_VBF_WW_inclu/BulkGraviton_VBF_WW_inclu_narrow_M2000/BulkGraviton_VBF_WW_inclu_narrow_M2000_run_card.dat [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+#*********************************************************************
+# Beam polarization from -100 (left-handed) to 100 (right-handed)    *
+#*********************************************************************
+        0     = polbeam1 ! beam polarization for beam 1
+        0     = polbeam2 ! beam polarization for beam 2
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+ 'lhapdf'  = pdlabel     ! PDF set                                     
+ 263400    = lhaid     ! if pdlabel=lhapdf, this is the lhapdf number
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/pre2017/14TeV/BulkGraviton_VBF_WW_inclu/BulkGraviton_VBF_WW_inclu_narrow_M2000/BulkGraviton_VBF_WW_inclu_narrow_M2000_run_card.dat]: you should use
```
 $DEFAULT_PDF_SETS = lhaid
 $DEFAULT_PDF_MEMBERS  = reweight_PDF

```
cfr https://twiki.cern.ch/twiki/bin/view/CMS/QuickGuideMadGraph5aMCatNLO#PDF_Choice_for_2017_production

File: bin/MadGraph5_aMCatNLO/cards/production/pre2017/14TeV/BulkGraviton_VBF_WW_inclu/BulkGraviton_VBF_WW_inclu_narrow_M2000/BulkGraviton_VBF_WW_inclu_narrow_M2000_proc_card.dat (hunk 1 above)
Comment (jalimena) [reply bin/MadGraph5_aMCatNLO/cards/production/pre2017/14TeV/BulkGraviton_VBF_WW_inclu/BulkGraviton_VBF_WW_inclu_narrow_M2000/BulkGraviton_VBF_WW_inclu_narrow_M2000_proc_card.dat]: Ok, removed these two definitions.

File: bin/MadGraph5_aMCatNLO/cards/production/pre2017/14TeV/BulkGraviton_VBF_WW_inclu/BulkGraviton_VBF_WW_inclu_narrow_M2000/BulkGraviton_VBF_WW_inclu_narrow_M2000_run_card.dat (hunk 2 above)
Comment (jalimena) [reply bin/MadGraph5_aMCatNLO/cards/production/pre2017/14TeV/BulkGraviton_VBF_WW_inclu/BulkGraviton_VBF_WW_inclu_narrow_M2000/BulkGraviton_VBF_WW_inclu_narrow_M2000_run_card.dat]: But these cards are to produce a signal for the HGCal TDR. My understanding was that we were supposed to keep the pdfs the same as in the previous TDRs, and not update them.

Comment (perrozzi) [reply bin/MadGraph5_aMCatNLO/cards/production/pre2017/14TeV/BulkGraviton_VBF_WW_inclu/BulkGraviton_VBF_WW_inclu_narrow_M2000/BulkGraviton_VBF_WW_inclu_narrow_M2000_run_card.dat]: My bad!

**Pull request merged**